Update project website

lily983 opened the pull request:
- Add features of PRIMP
- Add Delaware for Prof
- Upload video to YouTube(doing by Meng Xin)

ruansp commented on index.md:
```diff
@@ -1,21 +1,54 @@
+### Robot-agnostic
+- Skills can be easily transferred to another robot
+<p align="center">
+<img src="https://github.com/ChirikjianLab/primp-page/assets/71541515/49382079-edd6-4c35-9731-4451c08c3799" alt="robot agnostic" width="400" />
```
The same figure with adaptation to via points?

lily983 replied:
OK. I have changed the link, but I think I didn't successfully save them. I will update figure links.

ruansp commented on index.md:
```diff
@@ -1,21 +1,54 @@
+### Adaptation to unseen scenarios
+- Avoid novel obstacles while keep the motion features
+<p align="center">
+<img src="https://github.com/ChirikjianLab/primp-page/assets/71541515/49382079-edd6-4c35-9731-4451c08c3799" alt="avoid obs" width="400" />
```
Also the same figure?

ruansp commented on index.md:
```diff
@@ -1,21 +1,54 @@
+</p>
+
+### Adaptation to unseen scenarios
+- Avoid novel obstacles while keep the motion features
```
-> "Avoid previously unseen obstacles while keeping the key features"

lily983 commented:
Hi Sipu, I have fixed the figure link errors and added more explanation to sub-titles. BTW, do we need to update the paper on Arxiv? The current version is not up-to-date.

ruansp commented:
> Hi Sipu, I have fixed the figure link errors and added more explanation to sub-titles. BTW, do we need to update the paper on Arxiv? The current version is not up-to-date.

I don't think it is necessary since our paper in IEEE Xplore is open access. Also better not to be the same with the published version, which might cause copyright issues. I have linked the journal reference using DOI.
